README.md

```diff
@@ -47,7 +46,7 @@ many custom contracts.
 If you don't know what CosmWasm is, please check out
 [our homepage](https://cosmwasm.com) and
 [our documentation](https://docs.cosmwasm.com) to get more background.
-We are running a [public testnet](https://github.com/CosmWasm/testnets/blob/master/coralnet/README.md)
+We are running a [public testnet](https://github.com/CosmWasm/testnets/blob/master/sandynet-1/README.md)
```

Comment:
Thanks for this update

Reply:
Thanks for parsing through the linter junk to see the 1 liner. I'll look out for linter junk on future PRs.

Reply:
Can you update this PR to be that one-line change? (So you can get the commit credit)

Happy for a follow up to ensure consistent linting, but it must be a script, not just your IDE (and mine is different)